### PR TITLE
Add monitored item wrapper tests

### PR DESCRIPTION
**Pull request by simonjaeger**
Adding tests to ensure that the properties and filters are set correctly from the respective templates; `DataMonitoredItemModel `and `EventMonitoredItemModel`. 

To do this, the access modifier of the `MonitoredItemWrapper` class and its `Create` method had to be changed. The `Session `parameter was removed only to bring the two parameters needed; `MessageContext `and `TypeTree`.

### CODE REVIEW

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs [hunk 1]
```diff
@@ -1022,24 +1022,25 @@ private class MonitoredItemWrapper {
...
             /// Create new stack monitored item
             /// </summary>
-            /// <param name="session"></param>
+            /// <param name="messageContext"></param>
+            /// <param name="typeTree"></param>
             /// <param name="codec"></param>
             /// <param name="activate"></param>
             /// <returns></returns>
-            internal void Create(Session session, IVariantEncoder codec, bool activate) {
+            public void Create(ServiceMessageContext messageContext, ITypeTable typeTree, IVariantEncoder codec, bool activate) {
```
Comment (peterbryntesson) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:1030]: In the simplified event configuration I need access to the NodeCache, so use NodeCache as the parameter instead

Comment (simonjaeger) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:1030]: Good catch! I will fix.

Comment (buzzfrog) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:1030]: Is it better to name the parameter typeTable to reflect interface name

Comment (simonjaeger) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:1030]: Agreed.

Comment (simonjaeger) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs]: Now resolved.

**Review by mregen — APPROVED** [review]

**Pull request merged**